docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-  p_{e_1,e_2} de_1 de_2\\
-  \end{align*}`
+Risks with effects on the same outcome
+--------------------------------------
 
-.. math::
-  :nowrap:
+When we introduce correlation between two risk factors that have effects on the same outcome, we have to
+be especially careful about double-counting issues.
 
-  \begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2
-  \end{align*}
+The population-attributable fractions (PAFs) must be calculated correctly,
+or the population-level incidence of the outcome will
+not validate to GBD.
+**The usual Vivarium strategy to calculate joint PAFs (PAFs of more than one risk factor on an outcome)
+is biased when the risk factors are correlated.**
 
-Dividing through and cancelling terms yields a solution:
+The true population attributable fraction for the combined effect of two risk factors,
+:math:`r1` and :math:`r2`, on an outcome :math:`O` such as a mortality rate is (assuming no effect modification, as we typically do):
 
 .. math::
-
-
-   \frac{1}{1-\text{PAF}} = \int_{e_1} \int_{e_2} \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-   p_{e_1,e_2} de_1 de_2,
-
-.. code:: python
-
-    # there are lots of ways to approximate this integral, but here is a cool one:
-    rr_1 = 1.5
-    rr_2 = 3
-
-    e1 = np.clip((df['SBP'] - 120)/10, 0, np.inf)  # rescale exposures
-    e2 = np.clip((df['LDL-C'] - 4), 0, np.inf)
-
-    one_over_one_minus_paf = np.mean(rr_1**e1 * rr_2**e2)
-    paf = 1 - 1/(one_over_one_minus_paf)
-    paf
-
-
-
-
-.. parsed-literal::
-
-    0.936944660107624
-
-
-
-.. code:: python
-
-    # how does this compare to multiplicative approx of paf?
-    ooomp_1 = np.mean(rr_1**e1)
-    paf_1 = 1 - 1/ooomp_1
-
-    ooomp_2 = np.mean(rr_2**e2)
-    paf_2 = 1 - 1/ooomp_2
-
-    1 - (1 - paf_1) * (1 - paf_2)
-
-
-
-
-.. parsed-literal::
-
-    0.8708098034918508
-
-
-
-These values are somewhat different, reflecting the impact of the
-correlation in SBP and LDL-C in our simulated data. If we used the
-*multiplicative* PAF, our estimated rate would be quite different. Let’s
-look at a simpler model of categorical risks.
-
-Here is an example of categorial risk exposures
-=======================================================
-
-.. code:: python
-
-    # simulate categorical data (to be replaced with real data, e.g. from MALED/Surveys, eventually)
-    df = pd.DataFrame(index=range(N))
-    # Uncorrelated, binomial
-    df['bin_a'] = np.random.binomial(1, 0.3, size=N)
-    df['bin_b'] = np.random.binomial(1, 0.1, size=N)
-
-    rr_1 = 2.5
-    rr_2 = 5
-
-
-
-Define what we will call a joint PAF:
+  O = O_{tmrel} * RR_{r1} * RR_{r2}
```

Comment:
Just a comment that I'm pretty sure this equation is where the causality and no confounding assumptions are needed when dealing with PAFs and RRs.

Namely, O_{tmrel} is the outcome in the *counterfactual* scenario, while O is the outcome in the *factual* scenario. In order to relate them via this equation, we need the assumption that the the joint relative risk RR(r1,r2) is causal (hence equal to the population RR in the counterfactual scenario, I think) and equals the population relative risk in the factual scenario (which I think is equivalent to no confounding of the factual population's relative risks).

There's nothing to add or change here, I just thought I'd point this out since I've been thinking about it, and see if anyone wants to disagree.